client/state/states-list/schema.js

```diff
@@ -0,0 +1,17 @@
+export const statesListSchema = {
```

Comment:
Thanks for that!
In terms of tests, I think [these](https://github.com/Automattic/wp-calypso/blob/2cedce86d1250a54fe011f1438415265b5147655/client/state/states-list/test/reducer.js#L76-L91) should cover it?